src/axom/quest/SignedDistance.hpp

```diff
@@ -643,23 +642,22 @@ inline void SignedDistance<NDIMS, ExecSpace>::checkCandidate(
 
     if(shouldUpdateNormals)
     {
-      VectorType norm = surface_elems[ei].normal();
       ++currMin.minCount;
 
       switch(cpt_type)
       {
       case detail::ClosestPointLocType::edge:
-        // Candidate closest point is on an edge - add the normal of a
-        // potentially-adjacent face
-        currMin.sumNormals += norm;
+        // Add the (unitized) normal of the (potentially) incident face along this edge
+        // Note: Unitizing is important in cases where the edge is shared
+        // by triangles with different areas and an acute dihedral angle
+        currMin.sumNormals += surface_elems[ei].normal().unitVector();
```

Comment:
The bugfix is in this line:
* We have to use the unit normal vector for edges to account for possible differences in the triangle areas
* We were already using unit normals for vertices, as can be seen a few lines down